tests/test_choices.py

```diff
@@ -27,3 +27,11 @@ def test_snarky():
 
     assert len(choices) == 20
     assert provider.query(None, None) in choices
+
+
+def test_classic():
```

Comment:
m8, you're redefining the same function name :)

```suggestion
def test_spooky():
```